apartmenttherapy/order-service
# Elaine/bug updatedAt

#### What does this PR do & why?
Fixes a bug where updating an order was not updating the `updatedAt` field. Oops, this basically never worked.

#### How can this be manually tested?
- Tests should run.
- Update an order; the `updatedAt` field should update to the current timestamp.

#### Any questions or concerns about this change?
*(Any significant changes you're introducing? What side effects might this have? What other parts of the app could this affect?)*


#### Deployment Notes
*(e.g. Does this require a migration or special infrastructure change? Clearing the cache? Third party service?)*


#### Link to story card and other documentation
https://app.asana.com/0/694906336753887/1123896892309676

#### GIF


#### Have you remembered to...?
